variants/XMC1400/config/XMC1400_Arduino_Kit/pins_arduino.h

```diff
@@ -0,0 +1,290 @@
+  version 2.1 of the License, or (at your option) any later version.
+
+  This library is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+  Lesser General Public License for more details.
+
+  You should have received a copy of the GNU Lesser General
+  Public License along with this library; if not, write to the
+  Free Software Foundation, Inc., 59 Temple Place, Suite 330,
+  Boston, MA  02111-1307  USA
+*/
+#ifndef PINS_ARDUINO_H_
+#define PINS_ARDUINO_H_
+
+//****************************************************************************
+// @Project Includes
+//****************************************************************************
+#include <XMC1400.h>
+
+//****************************************************************************
+// @Defines
+//****************************************************************************
+// XMC_BOARD for stringifying into serial or other text outputs/logs
+// Note the actual name XMC and number MUST have a character between
+// to avoid issues with other defined macros e.g. XMC1400
+#define XMC_BOARD           XMC 1400 Kit for Arduino
+
+/* On board LED is ON when digital output is 0, LOW, FALSE, OFF */
+#define  XMC_LED_ON         0
+
+// Following were defines now evaluated by compilation as const variables
+// After definitions of associated mapping arrays
+extern const uint8_t NUM_DIGITAL;
+extern const uint8_t GND;
+extern const uint8_t NUM_PWM4;
+extern const uint8_t NUM_PWM;
+extern const uint8_t NUM_INTERRUPT;
+extern const uint8_t NUM_ANALOG_INPUTS;
+#define NUM_LEDS            3
+#define NUM_SERIAL          1
+#define NUM_TONE_PINS       4
+#define NUM_TASKS_VARIANT   8
+
+// Indicate unit has RTC/Alarm
+#define HAS_RTC             1
+
+// Defines will be either set by ArduinoIDE in the menu or manually
+#ifdef SERIAL_HOSTPC
+// Comment out following line to use Serial on pins (board)
+#define SERIAL_DEBUG    1
+#elif SERIAL_ONBOARD
+// No SERIAL_DEBUG will be defined, kept here for clarity
+#else
+// Define the SERIAL_DEBUG as default setting
+#define SERIAL_DEBUG    1
+#endif
+
+#define PWM4_TIMER_PERIOD (2041U)  // Generate 490Hz @fCCU=1MHz
+
+#define PCLK 96000000u //PCLK can go to this max value
+
+#define PIN_SPI_SS    10
+#define PIN_SPI_MOSI  11
+#define PIN_SPI_MISO  12
+#define PIN_SPI_SCK   13
+
+#define A0   0
+#define A1   1
+#define A2   2
+#define A3   3
+#define A4   4
+#define A5   5
+
+#define AD_AUX_1    6  // AD_AUX
+#define AD_AUX_2    7  // AD_AUX
+#define AD_AUX_3    8  // AD_AUX
+#define AD_AUX_4    9  // AD_AUX
+#define AUX_1       26  // AUX
+
+#define LED1        13
+#define LED2        26
+#define LED3        27
+#define LED_BUILTIN LED1
+
+#define EXT_INTR_0  3
+#define EXT_INTR_1  26
+
+#define digitalPinToInterrupt(p)    ((p) == 3 ? 0 : ((p) == 26 ? 1 : NOT_AN_INTERRUPT))
```

Comment:
Int 0 and 1 should be on pins 2 and 3

Reply:
The interrupts had to named differently as mentioned above.